proto/enums/enums.proto

```diff
@@ -419,6 +432,12 @@ enum StatusCode {
     INSPECT_BUILD_ENV_INCORRECT_TYPE_ERR = 1303;
     // Trying to modify a profile that doesn't exist
     INSPECT_PROFILE_NOT_FOUND_ERR = 1304;
+
+    // Kptfile errors
+    // The Kptfile is not a valid yaml file
+    KPTFILE_INVALID_YAML_ERR = 1401;
+    // The Kptfile is not a valid API schema
+    KPTFILE_INVALID_SCHEMA_ERR = 1402;
```

Comment:
Can you also add the relevant phase to this error?
These happen in "Deploy" phase right?

The CC error tracking uses the "Prefix" to reverse map the phase to the error code.

Reply:
Sure. Added. The function will be called by Runner.Render, so added to "render" phase.